pyam_analysis/core.py

```diff
@@ -675,7 +697,8 @@ def filter(self, filters, inplace=False):
             names=('model', 'scenario')
         )
         ret._meta = ret._meta.loc[idx]
-        return ret
+        if not inplace:
```

Comment:
why was this added?

Reply:
because pandas logic says `return None` if `inplace=True`

Reply:
Yes, good catch... I just read way too much why they do it. I don't like it, but they have good reasons.